### DISCUSSION TITLE
A custom http_server parameter in the rx.Config (rxconfig.py)

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [reflex-dev/reflex#3561] [Ideas]
Can we use a custom ASGI server for the Reflex app (**YES, WE CAN**)
but what if we add a parameter so that we don't need to do this
```bash
# Running these 2 commands separately.
granian --interface asgi my_app:app 
reflex run --frontend-only
```
INSTEAD, WE COULD DO THIS 🔽 AND RUN THE REFLEX APP WITH `reflex run`
(**Let's say, this is my rxconfig.py file**)
I've decided to use a fast web server for a better performance
```python
import reflex as rx

config = rx.Config(
    app_name="cherrysaas",
# Using a custom HTTP server here like this
   http_server="granian --interface asgi" # This is evaluated as "granian --interface asgi" + " <app_name>" (app_name not to be included in server)
)
```
Since it is not possible to spawn separate two shells for 2 commands in the REFLEX DEPLOY (reflex.run). This will become a problem for me to deploy using a custom HTTP server like granian. If you find my idea is legit, please convert this to an issue. **Somehow using a different HTTP server can boost FastAPI performance🤔**

**Comment by itsmeadarsh2008** [comment]
**Fun fact**: I still couldn't ever figure out which my_app in the granian --interface asgi my_app:app  (I mean which file it was pointing to)

**Comment by KronosDev-Pro** [comment]
#4337
